### PR TITLE
adds waiting state for buttons and mocks in a test into sotyrbook, re…

### PR DESCRIPTION
**Pull request by stuartromanek**
…factors to button component to be named in line with other AposComponents

### CODE REVIEW

**Comment by abea** [comment]
As a reviewer convenience, it's nice to have file renaming in its own commit with no other changes, so the diff can be easier to read. Github missed the renaming on this one.

File: src/scss/_theme.scss [hunk 1]
```diff
@@ -70,6 +70,13 @@ $boolean-padding: $spacing-double;
 
   // Universal
   --a-white: #fff;
+
+  // Apostrophe brand
```
Comment (abea) [inline src/scss/_theme.scss]: You didn't use these here. It'd be better to add them in a separate commit or in one where you are using them.

File: src/components/loading/AposLoading.vue [hunk 2]
```diff
@@ -0,0 +1,146 @@
+<template>
+  <div class="apos-loading">
```
[the remark this answers is not in the document]
Comment (abea) [reply src/components/loading/AposLoading.vue]: Wow.

File: src/components/button/AposButton.stories.js [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+    components: { AposButton },
+    template: '<AposButton @click="fakeBusy" v-bind:busy="busy" label="Filter" />',
+    ...mixin
+  }))
+  .add('Default, Disabled', () => ({
+    components: { AposButton },
+    template: '<AposButton @click="fakeBusy" label="Filter" v-bind:disabled="true" />',
+    ...mixin
+  }))
+  .add('Default, Busy', () => ({
```
Comment (abea) [inline src/components/button/AposButton.stories.js]: Let's do this for primary and danger as well since the black spinner won't work there.

Comment (stuartromanek) [reply src/components/button/AposButton.stories.js]: Not sure what you mean?

File: src/components/loading/AposLoading.vue (hunk 2 above)
[the remark this answers is not in the document]
Comment (stuartromanek) [reply src/components/loading/AposLoading.vue]: wow animated svg rocks, think this is my first go

File: src/components/button/AposButton.stories.js (hunk 3 above)
Comment (abea) [reply src/components/button/AposButton.stories.js]: Adding variations for those color backgrounds.
